Makefile

```
@@ -9,6 +9,9 @@ BUILDDIR      = build
 LINTER        = doc8
 LINTEROPTS    = --ignore D001 # D001 is linelength
 LANGMAP       = es_MX: es, fr_CA: fr, he_IL: he, tr_TR: tr
+SIZECHECKER   = python scripts/imagesizechecker.py
+SIZEMAX       = 300
```

```suggestion
SIZEMAX       = 500
```
Should this be `500`?